mnefun/_mnefun.py

```diff
@@ -1699,7 +1728,7 @@ def gen_forwards(p, subjects, structurals, run_indices):
                 if not op.isfile(trans):
                     raise IOError('Unable to find head<->MRI trans file')
             src_space_file = op.join(subjects_dir, structurals[si], 'bem',
-                                     structurals[si] + '-oct-6-src.fif')
+                                     structurals[si] + '-oct6-src.fif')
```

Comment:
Argh, need to revert/adapt this

Reply:
I am pretty sure we use `struc = structurals[si]` in `gen_forwards`. Any reason why we can't substitute thereafter?
Otherwise LGTM

Reply:
Works to fix https://github.com/LABSN/mnefun/issues/195 for me